swap/swap.go

```diff
@@ -532,6 +531,11 @@ func (s *Swap) verifyContract(ctx context.Context, address common.Address) error
 	return contract.ValidateCode(ctx, s.backend, address)
 }
 
+// SetChequebookAddr sets the chequebook address
+func (s *Swap) SetChequebookAddr(chequebookAddr common.Address) {
```

Comment:
Any reason why this needs to be exported?
It was `s.owner.Contract = <addr>`, thus it was not possible to set it from outside the package, now it would be. Not that I see a real security threat, but I'd like to know why you want to export it.

Reply:
Please attribute this to my relative inexperience with Golang. With hindsight, it is indeed not needed to export it. Why I thought differently when I made this change: no idea anymore. Thanks for the catch!